### PR TITLE
Pce watchdog tests

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

**Comment by mkopec** [comment]
All tests pass on coreboot rev https://github.com/Dasharo/coreboot/commit/7cf3914d2cc79b6aa6e48e59a784b6c16c85a4e4

**Review by macpijan — COMMENTED** [review]
@mkopec sorry I have just notice i have not pressed `submit review`. Please take a look if this still applies.

File: dasharo-compatibility/apu-configuration-menu.robot [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+*** Test Cases ***
+APU001.001 Check if apu2 watchdog option is available
+    [Documentation]    Check if the watchdog timer can be enabled in the apu2
+    ...    configuration submenu.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    APU001.001 not supported
+    Skip If    not ${APU_CONFIGURATION_MENU_SUPPORT}    APU configuration tests not supported.
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${apu_menu}=    Enter Dasharo APU Configuration    ${setup_menu}
+    ${index}=    Get Index Of Matching Option In Menu    ${apu_menu}    Enable watchdog
```
[the remark this answers is not in the document]
Comment (mkopec) [reply dasharo-compatibility/apu-configuration-menu.robot:37]: `Should Contain` looks for an exact match so it'd have to be `Enable watchdog [ ]` or `Enable watchdog [X]`. Unless there's a way to use a wildcard?

[the remark this answers is not in the document]
Comment (mkopec) [reply dasharo-compatibility/apu-configuration-menu.robot:37]: Okay, should contain match works.

[the remark this answers is not in the document]
Comment (mkopec) [reply dasharo-compatibility/apu-configuration-menu.robot]: 35ee869cd4c0b1e930fe2d7d7c6321eff4f9a1fa

File: dasharo-compatibility/apu-configuration-menu.robot [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    ...    not reset the platform anymore.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    APU002.001 not supported
+    Skip If    not ${APU_CONFIGURATION_MENU_SUPPORT}    APU configuration tests not supported.
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${apu_menu}=    Enter Dasharo APU Configuration    ${setup_menu}
+    ${current_profile}=    Set Option State    ${apu_menu}    Enable watchdog
+    ...    ${FALSE}
+    Save Changes And Reset
+    Enter Setup Menu Tianocore
```
[the remark this answers is not in the document]
Comment (mkopec) [reply dasharo-compatibility/apu-configuration-menu.robot]: Yes: 35ee869cd4c0b1e930fe2d7d7c6321eff4f9a1fa

**Review by macpijan — APPROVED** [review]
LGTM. Let's await for confirmation from @pkubaj that it works on another unit as well.

**Comment by macpijan** [comment]
@pkubaj 

> LGTM. Let's await for confirmation from @pkubaj that it works on another unit as well.

Have you checked something?

**Comment by pkubaj** [comment]
Just checked, seems to work.

**Pull request merged**